src/MooLite/plugins/EquipmentExporter/EquipmentExporterPlugin.ts

```diff
@@ -0,0 +1,49 @@
+import { markRaw } from "vue";
+import { MooLitePlugin } from "src/MooLite/core/plugins/MooLitePlugin";
+import { MooLiteTab } from "src/MooLite/core/plugins/MooLiteTab";
+import EquipmentExporterPluginDisplay from "src/MooLite/plugins/EquipmentExporter/EquipmentExporterPluginDisplay.vue";
+import EquippedItems from "src/MooLite/core/inventory/items/EquippedItems";
+import {CharacterAbility} from "src/MooLite/core/abilities/CharacterAbility";
+
+export class EquipmentExporterPlugin extends MooLitePlugin {
+    name: string = "Equipment Exporter";
+    key = "equipment-exporter";
+    description: string = "Export your equipment details for the Combat Sim";
+
+    tab: MooLiteTab = {
+        icon: "🗡",
+        pluginName: this.name,
+        componentName: "EquipmentExporterPluginDisplay",
+        component: markRaw(EquipmentExporterPluginDisplay),
+    };
+
+    getEquipmentSummary(): Array<EquippedItems> | null {
+        const equippedItems = new Array<EquippedItems>();
+        const abilities = this._game.combat.getCombatUnit().combatAbilities;
+        const combatFood = this._game.combat.getCombatUnit().combatConsumables;
+        const allItems = this._game.inventory._characterItems;
+        ["head", "body", "legs", "feet", "hands", "main_hand", "two_hand", "off_hand", "pouch", "neck", "earrings", "ring"].forEach((location) => {
```

Comment:
You can grab these from the `ItemLocationTypeMap` so they stay in sync with game updates automatically

Reply:
I think you meant the ItemLocationDetailMap, which was commented out, I implemented it and it seems to be working fine.